doc/policies/conditions.rst

```diff
@@ -0,0 +1,128 @@
+.. _policy_conditions:
+
+Policy conditions
+-----------------
+
+Since privacyIDEA 3.1, *policy conditions* allow to define more advanced rules
+for policy matching, i.e. for determining which policies are valid for a
+specific request.
+
+Conditions can be added to a policy via the WebUI. In order for a policy to
+take effect during the processing of a request, the request has to match not
+only the ordinary policy attributes (see :ref:`policies`), but also *all*
+additionally defined conditions that are currently active. If no active
+conditions are defined, only the ordinary policy attributes are taken into
+account.
+
+Each policy condition performs a comparison of two values. The left value is
+taken from the current request. The comparison operator (called *Comparator*)
+and the right value are entered in the policy definition. Each condition
+consists of five parts:
+
+ * ``Active`` determines if the condition is currently active.
+ * ``Section`` refers to an aspect of the incoming request on which the condition is formulated.
+   The available sections are predefined, see `Sections`_.
+ * The meaning of ``Key`` depends on the chosen ``Section``. Typically, it determines the exact property
+   of the incoming request on which the condition is formulated.
+ * ``Comparator`` defines the comparison to be performed. The available comparators are predefined, see `Comparators`_.
+ * ``Value`` determines the value the property should be compared against.
+
+Sections
+~~~~~~~~
```

Comment:
SHould we say, that currently there is only the section *userinfo*?

Reply:
Done in 4c03d0e7